governator-core/src/main/java/com/netflix/governator/conditional/Conditional.java

```diff
@@ -0,0 +1,37 @@
+package com.netflix.governator.conditional;
+
+/**
+ * Contract for any conditional that may be applied to a conditional binding
+ * bound via {@link ConditionalBinder}.
+ * 
+ * Dependencies needed by a concrete conditional should be injected using 
+ * member injection.
+ * 
+ */
+public abstract class Conditional {
```

Comment:
Ah, could we extract the part that is the 3 interface methods then and turn Conditional into an enum with just the static method?